src/core/agent/services/connectionService.ts

```diff
@@ -167,8 +168,14 @@ class ConnectionService extends AgentService {
 
   @OnlineOnly
   async deleteConnectionById(id: string): Promise<void> {
+    await this.props.signifyClient
+      .contacts()
+      .delete(id)
+      .catch((message) => {
+        throw message;
+      });
+
     await this.connectionStorage.deleteById(id);
-    // await this.signifyApi.deleteContactById(id); @TODO - foconnor: Uncomment when KERIA endpoint fixed
```

Comment:
I think we can keep this as it was but move the line up - so still commented out and with the TODO

Reply:
Is this the only case in the repository where this happens?

Reply:
yes, that's all I found
Do we update signify-ts before merging this PR? Because the version we use has looks like there is no PR for this yet https://github.com/WebOfTrust/signify-ts/pull/264

Reply:
We can't update Signify-TS because my IPEX PR isn't merged yet and it's blocked because of integration test failures.

We don't need to update it in this PR - just move the commented out line one line up. That should be enough for this entire PR if that is the only case you found